coralogix-policies/coralogix-metrics-integration-policy/template.yaml

```diff
@@ -114,6 +114,8 @@ Resources:
                   - ecs:DescribeServices
                   - ecs:ListContainerInstances
                   - ecs:DescribeContainerInstances
+                  -	elasticache:DescribeCacheClusters
```

Comment:
The indentation looks wrong

Reply:
I do not see it maybe globally it is wrong?

Reply:
Found tabs locally, did you notice in gihub UI or you review in editor? 
I have seen it in editor straight away 😮‍💨